src/ci/bin/testCentaurAws.sh

```diff
@@ -55,6 +55,9 @@ cromwell::build::run_centaur \
     -e continue_on_return_code \
     -e globbingscatter \
     -e inline_file_custom_entryname \
-    -e draft3_globs   
+    -e draft3_globs \
+    -e relative_output_paths \
+    -e relative_output_paths_colliding \
+    -e standard_output_paths_colliding_prevented \
```

Comment:
```suggestion
    -e standard_output_paths_colliding_prevented 
```

Reply:
AWS passed both PR & push, so the trailing slashes by themselves weren't the problem... 🤔

Reply:
AWS is also the only PR with all the `-e`s grouped, but I can't see from the code why not grouping `-e`s would be a problem

Reply:
Latest force-push gives up on cleanup / ordering. I'll figure out post release why the generated  `-l` options disappear when the `-e` options are passed in a different order.